### ISSUE TITLE
[SEMANTIC MARKUP] VAOS Appointment Cards - Bold section-heading like text must be marked up as headings

### ISSUE THREAD

**Issue by 1Copenut** [department-of-veterans-affairs/va.gov-team#16293]
# [508-defect-2](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/guidance/defect-severity-rubric.md#508-defect-2)

## Feedback framework

- **❗️ Must** for if the feedback must be applied 
- **⚠️ Should** if the feedback is best practice 
- **✔️ Consider** for suggestions/enhancements 

## Definition of done

1. Review and acknowledge feedback.
1. Refactor the cards to use headings and text instead of description lists.
1. Accessibility specialist will close ticket after reviewing documented decisions / validating fix.

<hr/>

## Point of Contact

**VFS Point of Contact:** _Trevor_

## User Story or Problem Statement

The Upcoming, Past, and Express Care cards use `<dl>` markup to create relationships between appointment sections and the information contained in that section. As a screen reader user, I would like to have a semantic meaning assigned to these sections like "Contact Details" and "You shared these details about your concern".

## Details

The VA 508 office flagged our description lists with bold text as needing to have proper semantic headings. I did a quick analysis and headings cannot be nested in `<dt>` tags, so this will require some HTML refactoring to pass muster. Screenshots and code snippet below.

## Acceptance Criteria

- [ ] Refactored HTML validates
- [ ] No axe violations shown
- [ ] Headings are properly nested on all card types

## Environment

* https://staging.va.gov/health-care/schedule-view-va-appointments/appointments/
* https://staging.va.gov/health-care/schedule-view-va-appointments/appointments/past
* https://staging.va.gov/health-care/schedule-view-va-appointments/appointments/express-care

## WCAG or Vendor Guidance (optional)

* [Info and Relationships: Understanding SC 1.3.1](https://www.w3.org/TR/UNDERSTANDING-WCAG20/content-structure-separation-programmatic.html)
* [Failure of Success Criterion 1.3.1 due to using changes in text presentation to convey information without using the appropriate markup or text](https://www.w3.org/WAI/WCAG21/Techniques/failures/F2)

## Screenshots or Trace Logs

![Screen Shot 2020-11-17 at 2 41 05 PM](https://user-images.githubusercontent.com/934879/99450283-2dbecc00-28e6-11eb-9e97-c348c88d9b83.png)

---
![Screen Shot 2020-11-17 at 2 41 11 PM](https://user-images.githubusercontent.com/934879/99450309-3adbbb00-28e6-11eb-8a11-a564b982653c.png)

---
![Screen Shot 2020-11-17 at 2 50 24 PM](https://user-images.githubusercontent.com/934879/99449986-ba1cbf00-28e5-11eb-8195-370634070942.png)

**Comment by jenniemc** [comment]
Hi @1Copenut, is this what you have in mind for the section heading to have h4 tags?


![image](https://user-images.githubusercontent.com/54327023/99720138-4d3b2d80-2a7b-11eb-8b46-e2309d3645a6.png)

**Comment by 1Copenut** [comment]
@jenniemc Yes, this is what I was envisioning.
